plugins/lsp/default.nix

```
@@ -231,34 +260,37 @@ in {
     mkIf cfg.enable {
       extraPlugins = [pkgs.vimPlugins.nvim-lspconfig];
 
-      keymaps = let
-        mkMaps = prefix:
-          mapAttrsToList
-          (
-            key: action: let
-              actionStr =
-                if isString action
-                then action
-                else action.action;
-              actionProps =
-                if isString action
-                then {}
-                else filterAttrs (n: v: n != "action") action;
-            in {
-              mode = "n";
-              inherit key;
-              action.__raw = prefix + actionStr;
+      keymapsOnEvents = {
+        "LspAttach" = let
```

Could avoid indenting the lines below by doing something like this:
```suggestion
      keymapsOnEvents."LspAttach" = let
```